src/quartz/tasograph/tasograph.cpp

```diff
@@ -1504,25 +1667,13 @@ bool Graph::xfer_appliable(GraphXfer *xfer, Op op) const {
     std::vector<OpX *> output_opxs(num_output, nullptr);
     if (inEdges.find(op_) != inEdges.end()) {
       for (auto &e : inEdges.find(op_)->second) {
+        if (!((size_t)e.dstIdx < num_input)) {
+          std::cout << 1663 << std::endl;
```

Comment:
Is this a debug output? Feel free to merge it as is if it won't output a lot of traces for every run.

Reply:
Yep this is a debug output. I'll remove them.